hack/tag.sh

```diff
@@ -4,7 +4,7 @@
 #  \\\\\ Copyright 2024-present SPIKE contributors.
 # \\\\\\\ SPDX-License-Identifier: Apache-2.0
 
-VERSION="v0.14.6"
+VERSION="v0.15.0"
```

Comment:
Just bumping a patch version would be a disservice to this PR :D -- so a minor version increase is due.